lib/typescript.js

```diff
@@ -45,8 +41,12 @@ export default [
         },
       ],
       "@typescript-eslint/prefer-namespace-keyword": ["error"],
-      "@typescript-eslint/no-unused-vars": ["error"],
-      "@typescript-eslint/no-undef": "off",
+      "@typescript-eslint/no-unused-vars": [
```

Comment:
this too

Reply:
Would do the same as the suggestion I made above